src/app/onto-search/search/search.component.ts

```diff
@@ -0,0 +1,86 @@
+import {Component, ElementRef, EventEmitter, Input, OnInit, Output, TemplateRef, ViewChild} from '@angular/core';
+import {Observable} from 'rxjs';
+import {FormControl} from '@angular/forms';
+import {map, startWith} from 'rxjs/operators';
+import {MatChipInputEvent} from '@angular/material/chips';
+import {MatAutocompleteSelectedEvent, MatAutocompleteTrigger} from '@angular/material/autocomplete';
+import {SearchConfiguration} from 'src/app/onto-search/models/search-configuration';
+import {SearchModel} from 'src/app/onto-search/models/search-model';
+
+@Component({
+  selector: 'app-search',
+  templateUrl: './search.component.html',
+  styleUrls: ['./search.component.scss'],
+})
+export class SearchComponent implements OnInit {
+  @Input()
+  private states: Observable<SearchModel[]>;
+  private _states: SearchModel[];
+
+  @Input()
+  public workingTemplate: TemplateRef<any>;
```

Comment:
Not very meaningful name. Perhaps `OptionTemplate` | `AutocompleteOptionTemplate`